### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by Nikhil23121998** [peridigm/peridigm#263] [General]
this error is showing while running YAML file of tensile_test, please help me to resolve this issue.

/home/nikhil/peridigm/src/core/Peridigm_BoundaryCondition.cpp:226:

Terminate test that evaluated to true: nodeSets->find(nodeSetName) == nodeSets->end()

**** Error in DirichletBC::apply(), node set not found: NODELIST_1

terminate called without an active exception
[DESKTOP-FQMM6J7:00183] *** Process received signal ***
[DESKTOP-FQMM6J7:00183] Signal: Aborted (6)
[DESKTOP-FQMM6J7:00183] Signal code:  (-6)
[DESKTOP-FQMM6J7:00183] [ 0] /lib/x86_64-linux-gnu/libpthread.so.0(+0x14420)[0x7f7226f81420]
[DESKTOP-FQMM6J7:00183] [ 1] /lib/x86_64-linux-gnu/libc.so.6(gsignal+0xcb)[0x7f7226db300b]
[DESKTOP-FQMM6J7:00183] [ 2] /lib/x86_64-linux-gnu/libc.so.6(abort+0x12b)[0x7f7226d92859]
[DESKTOP-FQMM6J7:00183] [ 3] /lib/x86_64-linux-gnu/libstdc++.so.6(+0x9e911)[0x7f72271ae911]
[DESKTOP-FQMM6J7:00183] [ 4] /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa38c)[0x7f72271ba38c]
[DESKTOP-FQMM6J7:00183] [ 5] /lib/x86_64-linux-gnu/libstdc++.so.6(+0xaa3f7)[0x7f72271ba3f7]
[DESKTOP-FQMM6J7:00183] [ 6] /home/nikhil/trilinos/lib/libteuchoscore.so.13(_ZN7Teuchos28TestForTermination_terminateERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE+0xc4)[0x7f7227b424f4]
[DESKTOP-FQMM6J7:00183] [ 7] /home/nikhil/peridigm/bin/Peridigm(_ZN10PeridigmNS20DirichletIncrementBC5applyEN7Teuchos3RCPISt3mapINSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEESt6vectorIiSaIiEESt4lessIS9_ESaISt4pairIKS9_SC_EEEEERKdSM_+0x9de)[0x7f722971140e]
[DESKTOP-FQMM6J7:00183] [ 8] /home/nikhil/peridigm/bin/Peridigm(_ZN10PeridigmNS34BoundaryAndInitialConditionManager23applyBoundaryConditionsERKdS2_+0x120)[0x7f7229702ed0]
[DESKTOP-FQMM6J7:00183] [ 9] /home/nikhil/peridigm/bin/Peridigm(_ZN10PeridigmNS8Peridigm18executeQuasiStaticEN7Teuchos3RCPINS1_13ParameterListEEE+0x2130)[0x7f7229627160]
[DESKTOP-FQMM6J7:00183] [10] /home/nikhil/peridigm/bin/Peridigm(_ZN10PeridigmNS8Peridigm7executeEN7Teuchos3RCPINS1_13ParameterListEEE+0x1c3)[0x7f7229632493]
[DESKTOP-FQMM6J7:00183] [11] /home/nikhil/peridigm/bin/Peridigm(_ZN10PeridigmNS8Peridigm14executeSolversEv+0x1e7)[0x7f7229632b47]
[DESKTOP-FQMM6J7:00183] [12] /home/nikhil/peridigm/bin/Peridigm(main+0x1ca)[0x7f72295f01fa]
[DESKTOP-FQMM6J7:00183] [13] /lib/x86_64-linux-gnu/libc.so.6(__libc_start_main+0xf3)[0x7f7226d94083]
[DESKTOP-FQMM6J7:00183] [14] /home/nikhil/peridigm/bin/Peridigm(_start+0x2e)[0x7f72295f3f5e]
[DESKTOP-FQMM6J7:00183] *** End of error message ***
Aborted (core dumped)

**Comment by sahirbutt** [comment]
It looks like you don't have nodelist_1 defined in your mesh file. Did you modify the mesh file?
You can check for the nodesets in the mesh file using `trilinos_install_dir/bin/explore mesh_file.g` and then `NSETS`
